src/apps/frontend/services/auth.service.ts

```diff
@@ -10,4 +10,12 @@ export default class AuthService extends APIService {
     username,
     password,
   });
+
+  signup = async (
+    username: string,
+    password: string,
+  ): Promise<ApiResponse<AccessToken>> => this.apiClient.post('/accounts', {
```

Comment:
I am suprised backend is sending access token on account creation. It should not.

We don't need to change backend, but we want to make sure this function should not return anything

Reply:
okay sir